### PR TITLE
add team docs

### PR DESCRIPTION
**Pull request by mobuffon**
hope i was able to add everything correctly

### CODE REVIEW

File: paths/teams/add_project.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      -d '{"id":"a4b3c2d1"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase teams projects \
+    --account_id <account_id> \
+    --team_id <team_id> \
+    --data '{"id":"a4b3c2d1"}' \
+    --access_token <token>
+- lang: CLI v1
```
Comment (rlisowski) [inline paths/teams/add_project.yaml:42]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/add_project.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/projects" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X POST \
+      -d '{"id":"a4b3c2d1"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase teams projects \
```
Comment (rlisowski) [inline paths/teams/add_project.yaml:37]: I don't think this is correct. It should be `phrase team projects_create` I think. Did you run the CLI generator?

File: paths/teams/add_space.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- lang: CLI v2
+  source: |-
+    phrase teams spaces \
+    --account_id <account_id> \
+    --team_id <team_id> \
+    --data '{"id":"a4b3c2d1"}' \
+    --access_token <token>
+- lang: CLI v1
+  source: |-
+    phraseapp teams spaces create <account_id> <team_id> \
```
Comment (rlisowski) [inline paths/teams/add_space.yaml:44]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/add_space.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/spaces" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X POST \
+      -d '{"id":"a4b3c2d1"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase teams spaces \
```
Comment (rlisowski) [inline paths/teams/add_space.yaml:37]: Seems to be incorrect.

File: paths/teams/add_user.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- lang: CLI v2
+  source: |-
+    phrase teams users \
+    --account_id <account_id> \
+    --team_id <team_id> \
+    --data '{"id":"a4b3c2d1"}' \
+    --access_token <token>
+- lang: CLI v1
+  source: |-
+    phraseapp teams users create <account_id> <team_id> \
```
Comment (rlisowski) [inline paths/teams/add_user.yaml:44]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/add_user.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/users" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X POST \
+      -d '{"id":"a4b3c2d1"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase teams users \
```
Comment (rlisowski) [inline paths/teams/add_user.yaml:37]: Again seems wrong.

File: paths/teams/create.yaml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      -d '{"name":"German Translators"}' \
+      -H 'Content-Type: application/json'
+- lang: CLI v2
+  source: |-
+    phrase teams create \
+    --account_id <account_id> \
+    --data '{"name":""German Translators""}' \
+    --access_token <token>
+- lang: CLI v1
+  source: |-
```
Comment (rlisowski) [inline paths/teams/create.yaml:45]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/delete.yaml [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X DELETE
+- lang: CLI v2
+  source: |-
+    phrase teams delete \
+    --account_id <account_id> \
+    --id <id> \
+    --access_token <token>
+- lang: CLI v1
+  source: phraseapp team delete <account_id> <id>
```
Comment (rlisowski) [inline paths/teams/delete.yaml:33]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/index.yaml [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams" \
+      -u USERNAME_OR_ACCESS_TOKEN
+- lang: CLI v2
+  source: |-
+    phrase teams list \
+    --account_id <account_id> \
+    --access_token <token>
+- lang: CLI v1
+  source: phraseapp teams list <account_id>
```
Comment (rlisowski) [inline paths/teams/index.yaml:47]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/remove_project.yaml [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+      -X DELETE
+- lang: CLI v2
+  source: |-
+    phrase teams projects \
+    --account_id <account_id> \
+    --team_id <team_id> \
+    --id <id> \
+    --access_token <token>
+- lang: CLI v1
+  source: phraseapp teams projects delete <account_id> <team_id> <id>
```
Comment (rlisowski) [inline paths/teams/remove_project.yaml:35]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/remove_space.yaml [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+      -X DELETE
+- lang: CLI v2
+  source: |-
+    phrase teams spaces \
+    --account_id <account_id> \
+    --team_id <team_id> \
+    --id <id> \
+    --access_token <token>
+- lang: CLI v1
+  source: phraseapp teams spaces delete <account_id> <team_id> <id>
```
Comment (rlisowski) [inline paths/teams/remove_space.yaml:35]: CLI v1 will not generate automatically, it's better to mark it as unsupported.

File: paths/teams/remove_project.yaml [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/projects/:id" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X DELETE
+- lang: CLI v2
+  source: |-
+    phrase teams projects \
```
Comment (rlisowski) [inline paths/teams/remove_project.yaml:29]: Seems to be incorrect.

File: paths/teams/remove_space.yaml [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/spaces/:id" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X DELETE
+- lang: CLI v2
+  source: |-
+    phrase teams spaces \
```
Comment (rlisowski) [inline paths/teams/remove_space.yaml:29]: Seems to be incorrect.

File: paths/teams/remove_user.yaml [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+    "$ref": "../../responses.yaml#/429"
+x-code-samples:
+- lang: Curl
+  source: |-
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:team_id/users/:id" \
+      -u USERNAME_OR_ACCESS_TOKEN \
+      -X DELETE
+- lang: CLI v2
+  source: |-
+    phrase teams users \
```
Comment (rlisowski) [inline paths/teams/remove_user.yaml:29]: Seems to be incorrect.

File: paths/teams/remove_user.yaml [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+    --account_id <account_id> \
+    --team_id <team_id> \
+    --id <id> \
+    --access_token <token>
+- lang: CLI v1
+  source: phraseapp teams users delete <account_id> <team_id> <id>
```
Comment (rlisowski) [inline paths/teams/remove_user.yaml:35]: Mark as unsupported instead.

File: paths/teams/show.yaml [hunk 16]
```diff
@@ -0,0 +1,43 @@
...
+    curl "https://api.phrase.com/v2/accounts/:account_id/teams/:id" \
+      -u USERNAME_OR_ACCESS_TOKEN
+- lang: CLI v2
+  source: |-
+    phrase teams show \
+    --account_id <account_id> \
+    --id <id> \
+    --access_token <token>
+- lang: CLI v1
+  source: phraseapp team show <account_id> <id>
```
Comment (rlisowski) [inline paths/teams/show.yaml:43]: Again not true.

File: paths/teams/update.yaml [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+- lang: CLI v2
+  source: |-
+    phrase teams update \
+    --account_id <account_id> \
+    --id <id> \
+    --data '{"name":""German Translators""}' \
+    --access_token <token>
+- lang: CLI v1
+  source: |-
+    phraseapp team update <account_id> <id> \
```
Comment (rlisowski) [inline paths/teams/update.yaml:48]: Nope.

File: schemas/team_detail.yaml [hunk 18]
```diff
@@ -0,0 +1,54 @@
+---
+team_detail:
```
Comment (rlisowski) [inline schemas/team_detail.yaml:2]: I don't see any difference with `team`, please use `team` instead and remove this file.

File: paths/teams/add_project.yaml (hunk 2 above)
Comment (mobuffon) [reply paths/teams/add_project.yaml:37]: no, i haven't yet. Also i dont know how it works. I will look into that

File: schemas/team_detail.yaml (hunk 18 above)
Comment (mobuffon) [reply schemas/team_detail.yaml:2]: it is within the user. team details adds more info

File: paths/teams/add_project.yaml (hunk 2 above)
Comment (mobuffon) [reply paths/teams/add_project.yaml:37]: @rlisowski I took a look at the paths/spaces/add_project.yaml file which describes the same functionality and there it is done without the underscore. I think it is either correct like this or we should change it there as well

Comment (rlisowski) [reply paths/teams/add_project.yaml:37]: Spaces docs are wrong, I will fix them in separate PR. Thanks for pointing it out.

```
$ phrase spaces -h
Spaces API

Usage:
  phrase spaces [command]

Available Commands:
  create          Create a Space
  delete          Delete Space
  list            List Spaces
  projects_create Add Project
  projects_delete Remove Project
  projects_list   List Projects
  show            Get Space
  update          Update Space
```

File: schemas/team.yaml [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+      items:
+        "$ref": "./project.yaml#/project"
+    spaces:
+      type: array
+      items:
+        "$ref": "./space.yaml#/space"
+    users:
+      type: array
+      items:
+        "$ref": "./user.yaml#/user"
```
Comment (rlisowski) [inline schemas/team.yaml:27]:
```suggestion
        "$ref": "./user_preview.yaml#/user_preview"
```

File: schemas/team_detail.yaml (hunk 18 above)
Comment (rlisowski) [reply schemas/team_detail.yaml:2]: In that case, team.yml is wrong and should point to user_preview.

File: paths/teams/add_project.yaml (hunk 2 above)
Comment (rlisowski) [reply paths/teams/add_project.yaml:37]: @mobuffon I fixed spaces docs

Comment (mobuffon) [reply paths/teams/add_project.yaml:37]: okay, thanks

**Comment by rlisowski** [comment]
Just note that to release it we need to migrate CLI to phrase-go 1.0.7.

**Comment by mobuffon** [comment]
@rlisowski can you help me out with migrating? I have no clue how that works

**Comment by rlisowski** [comment]
> @rlisowski can you help me out with migrating? I have no clue how that works

I will do it, just keep it open for now.

**Comment by mobuffon** [comment]
thanks!

**Pull request merged**